Documentation/raspi-libgpiodv2.md

```diff
@@ -0,0 +1,99 @@
+# Use libgpiodv2 on a Raspberry Pi
+
+If you want to control GPIOs using libgpiod version 2 the library must be installed.
```

Comment:
> libgpiod version 2 the library must be installed

This can also be confusing, since the version 2 of the library can be confused with v1.1, or v1.6.4. (as you helpfully explained in the issue). I wonder if some of that info you logged in the issue would help having documented too.